.eslintrc

```diff
@@ -30,7 +30,7 @@
     "no-shadow": 2,
     "no-shadow-restricted-names": 2,
     "no-spaced-func": 2,
-    "no-trailing-spaces": 2,
+    "no-trailing-spaces": 0,
```

Comment:
I've got tired of this thing being buggy in some cases and finally disabled it in all my projects. Yes, we can increase file size due to unnecessary invisible symbols, but we definitely will not break anything with it.